src/Installers/Windows/RuntimeSiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should this target release/2.2?

Reply:
Not, site extensions target net461 by convention.